.flake8

```
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 100
+extend-ignore = W503, E203
```

```suggestion
extend-ignore = W503
```

Instead of ignoring this error, could you fix those errors?

---

Sure I will, there is a small mistake in my flake8 fix as well, wrong indentation. Will fix that as well

---

Black adds whitespace before : in slices, and flake8 complains about it.

Recomended configs for black:
[Compatible configuration files can be found here](https://github.com/psf/black/blob/main/docs/compatible_configs/)

This behaviour may raise E203 whitespace before ':' warnings in style guide enforcement tools like Flake8. Since E203 is not PEP 8 compliant, you should tell Flake8 to ignore these warnings. [More here](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#e203).

WDYT?

---

Oh, that makes sense.
Could you add a comment here on why we must ignore E203?